internal/pkg/cli/job_package.go

```diff
@@ -4,22 +4,41 @@
 package cli
 
 import (
+	"errors"
 	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
 
-	"github.com/spf13/cobra"
-
+	"github.com/aws/copilot-cli/internal/pkg/addon"
+	"github.com/aws/copilot-cli/internal/pkg/aws/sessions"
 	"github.com/aws/copilot-cli/internal/pkg/config"
+	"github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation"
+	"github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation/stack"
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
 	"github.com/aws/copilot-cli/internal/pkg/term/command"
 	"github.com/aws/copilot-cli/internal/pkg/term/prompt"
 	"github.com/aws/copilot-cli/internal/pkg/term/selector"
 	"github.com/aws/copilot-cli/internal/pkg/workspace"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
 )
 
 const (
 	jobPackageJobNamePrompt = "Which job would you like to generate a CloudFormation template for?"
 	jobPackageEnvNamePrompt = "Which environment would you like to package this stack for?"
 )
 
+var initPackageAddons = func(o *packageJobOpts) error {
+	addonsSvc, err := addon.New(o.name, "job")
+	if err != nil {
+		return fmt.Errorf("initiate addons service: %w", err)
```

Comment:
nit: 
```suggestion
		return fmt.Errorf("new addons client: %w", err)
```

Reply:
Should I change "addons service" to "addons client" elsewhere, too? I did find this wording confusing.

Reply:
If possible that would be nice!